functions.py

```diff
@@ -0,0 +1,109 @@
+import time
+import turtle
+
+score = 1
+score_a = 0
+score_b = 0
+player_a = "Left Player"
+player_b = "Right Player"
+
+
+def create_game_win(left_rocket, right_rocket):
+    win = turtle.Screen()
+    win.title("Game")
+    win.bgcolor("black")
+    win.setup(width=800, height=600)
+    win.tracer(0)
+    win.listen()
+    win.onkeypress(rocket_up(left_rocket), "w")
+    win.onkeypress(rocket_down(left_rocket), "s")
+    win.onkeypress(rocket_up(right_rocket), "Up")
+    win.onkeypress(rocket_down(right_rocket), "Down")
```

Comment:
I suppose `onkeypress` takes callable (function) as the first argument, not the function execution results.
I see 2 ways how you can solve it, using [partial](https://docs.python.org/3/library/functools.html#functools.partial) or [lambda](https://docs.python.org/3/tutorial/controlflow.html#lambda-expressions).